tests/pageobjects/assistify.page.js

```diff
@@ -115,7 +127,7 @@ class Assistify extends Page {
 		browser.pause(500);
 	}
 
-	createHelpRequest(topicName, message) {
+	createHelpRequest(topicName, message, requestTitle) {
```

Comment:
make sure you don't mess up other tests which use this functions.
Is it possible to call this function with only 2 parameters or do you have to add '' if no requestTitle is needed?

Reply:
we would need to the function with 'requestTitle' because you would need to create a sub-request based on the requestTitle for tests scenario and when you do not pass the requestTitle  it will be difficult figure out the request as it will get a random number.